reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>Not sure if this is because of the ReviewNB app, but the bullet point 2 is shifted down by a line. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>